client/components/widget/query/builder/query-builder-service.js

```diff
@@ -77,12 +79,12 @@ const Aggregation = explorer.components.widget.query.builder.Aggregation;
 /**
  * The QueryBuilder service transforms a query model into SQL.
  *
- * @param {!Angular.FilterService} $filter
+ * @param {!angular.$filter} $unused_filter
  * @constructor
  * @ngInject
  */
 explorer.components.widget.query.builder.QueryBuilderService = function(
-    $filter) {
+    $unused_filter) {
```

Comment:
This parameter looks like it should be removed.  As it's not being exposed as a member property, it can't be accessed externally.